From c46d35b45fb17eaac643094e0302031a4cb79cca Mon Sep 17 00:00:00 2001
From: Min He <min.he@intel.com>
Date: Fri, 31 May 2019 05:45:30 +0000
Subject: [PATCH 1400/1505] drm/i915/gvt: reduce time for get forcewake

This patch removed unnecessary forcewake get during gvt submission
path, which reduced cpu usage of gvt workload thread.

Signed-off-by: Min He <min.he@intel.com>
Reviewed-by: Zhao Yakui <yakui.zhao@intel.com>
---
 drivers/gpu/drm/i915/gvt/scheduler.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/gvt/scheduler.c b/drivers/gpu/drm/i915/gvt/scheduler.c
index c7a7b0583dd1..c6233fb61753 100644
--- a/drivers/gpu/drm/i915/gvt/scheduler.c
+++ b/drivers/gpu/drm/i915/gvt/scheduler.c
@@ -1040,8 +1040,7 @@ static int workload_thread(void *priv)
 	int ret;
 	long lret;
 	bool need_force_wake = IS_SKYLAKE(gvt->dev_priv)
-			|| IS_KABYLAKE(gvt->dev_priv)
-			|| IS_BROXTON(gvt->dev_priv);
+			|| IS_KABYLAKE(gvt->dev_priv);
 	DEFINE_WAIT_FUNC(wait, woken_wake_function);
 
 	kfree(p);
-- 
2.17.1

